src/test_utils/mongo/mod.rs

```diff
@@ -156,9 +128,11 @@ impl MongoFuzzer {
     }
 
     /// Adds a transaction to the collection of transactions with custom values.
-    pub fn add_custom_transaction(&mut self, builder: TransactionBuilder) -> Result<(), Box<dyn std::error::Error>> {
-        // Build a transaction using the provided builder and random byte size.
-        let transaction = builder.build(self.rnd_bytes_size)?;
+    pub fn add_custom_transaction(&mut self) -> Result<(), Box<dyn std::error::Error>> {
```

Comment:
we should probably remove this and just keep `add_random_transactions`, what do you think?

Reply:
agreed, fixed